src/liblibc/lib.rs

```diff
@@ -3912,6 +3912,9 @@ pub mod consts {
             pub const _SC_XBS5_ILP32_OFFBIG : c_int = 126;
             pub const _SC_XBS5_LPBIG_OFFBIG : c_int = 128;
 
+            // Linux
+            pub const _PC_NAME_MAX: c_int = 3;
+            pub const _PC_PATH_MAX: c_int = 4;
```

Comment:
Ah yeah it doesn't matter too much to me where these go so long as they're accessible